CMakeLists.txt

```diff
@@ -1782,8 +1782,40 @@ target_sources(${PROJECT_NAME} PRIVATE
         core/gdxsv/gdxsv_patch.inc
         core/gdxsv/gdxsv_emu_hooks.cpp
         core/gdxsv/gdxsv_emu_hooks.h
+		core/gdxsv/gdxsv_translation.h
+		core/gdxsv/gdxsv_translation.cpp
+		core/gdxsv/gdxsv_translation_patch_utf8.h
+		core/gdxsv/gdxsv_CustomTexture.h
+		core/gdxsv/gdxsv_CustomTexture.cpp
         core/gdxsv/lbs_message.h
         core/gdxsv/mcs_message.h)
+
+if(APPLE)
+		set(ASSETS core/gdxsv/Textures)
+		target_sources(${PROJECT_NAME} PRIVATE ${ASSETS})
+		source_group("Resources" FILES ${ASSETS})
+		set_source_files_properties(${ASSETS} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
+elseif(WIN32)
+		add_custom_target(gdxsv_translation_generate_texture ALL
```

Comment:
Why only win32?
The calling bash fail on MSVC and become build error.

Reply:
Seems the generated build from GitHub doesn't have any error and can generate the correct `core/gdxsv/textures.rc` file? (I haven't commit the generated files and the Windows build has the patched graphics)

Reply:
Not converted to Python yet, but verified the bash script can be run in Git Bash, `textures.rc` are committed now